tests/filecheck/dialects/riscv/add.mlir

```diff
@@ -0,0 +1,7 @@
+// RUN: xdsl-opt %s | xdsl-opt | filecheck %s
+"builtin.module"() ({
+  %lhs = "test.op"() : () -> !riscv.reg<>
+  %rhs = "test.op"() : () -> !riscv.reg<>
+  %sum = "riscv.add"(%lhs, %rhs) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
+// CHECK: %sum = "riscv.add"(%lhs, %rhs) : (!riscv.reg<>, !riscv.reg<>) -> !riscv.reg<>
```

Comment:
Do you want to test the SSA names here? I would advise against it personally... :)

Reply:
Do we expect it to break?